third_party/blink/renderer/modules/mediasource/media_source.cc

```diff
@@ -564,6 +533,14 @@ bool MediaSource::IsTypeSupportedInternal(ExecutionContext* context,
     return false;
   }
 
+#if BUILDFLAG(IS_COBALT)
+#if BUILDFLAG(USE_STARBOARD_MEDIA)
+  // Interupt Chromium's IsTypeSupported() from here for better performance.
+  SbMediaSupportType support_type = SbMediaCanPlayMimeAndKeySystem(
+      type.DeprecatedLower().Ascii().c_str(), "");
+  return support_type != kSbMediaSupportTypeNotSupported;
```

Comment:
Add LOG(INFO)

Reply:
Log added in function MediaSource::addSourceBuffer().